### PR TITLE
Simplify install instructions

### PR DESCRIPTION
**Pull request by arcondello**
...and remove the hardcoded python version reference.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -59,27 +59,12 @@ Installation
 
 .. installation-start-marker
 
-Compatible with Python 3.6+:
+Installation from [PyPI](https://pypi.org/project/dimod/):
```
Comment (JoelPasvolsky) [inline README.rst:62]:
```suggestion
Installation from `PyPI <https://pypi.org/project/dimod>`_:
```

This is RST format

**Review by JoelPasvolsky — COMMENTED** [review]
Why don't we want installation from source anymore?

**Comment by arcondello** [comment]
> Why don't we want installation from source anymore?

Under the assumption that anyone who wants to make a contribution probably understands `pip` and `setuptools`. If we see a lot of confusion, then we can restore it.

**Comment by JoelPasvolsky** [comment]
> > Why don't we want installation from source anymore?
> 
> Under the assumption that anyone who wants to make a contribution probably understands `pip` and `setuptools`. If we see a lot of confusion, then we can restore it.

okay. I used to look here for a reminder on the commands but it's not hard to find that elsewhere

**Review by JoelPasvolsky — APPROVED** [review]

**Pull request merged**